src/Backoffice/Cities/App/Controllers/StoreCityController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Cities\App\Controllers;
+
+use Lightit\Backoffice\Cities\App\Requests\StoreCityRequest;
+use Lightit\Backoffice\Cities\App\Transformers\CityTransformer;
+use Lightit\Backoffice\Cities\Domain\Actions\StoreCityAction;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class StoreCityController
+{
+    public function __invoke(StoreCityRequest $request, StoreCityAction $action): JsonResponse
+    {
+        $city = $action->execute($request->toDto());
+
+        return responder()
+            ->success([
+                'message' => 'City created',
+                'data' => $city,
+                CityTransformer::class,
+            ])
```

Comment:
```suggestion
            ->success($city,CityTransformer::class)
```
Just do this, flugg takes care of the rest. Same for the rest of the endpoints